### DISCUSSION TITLE
Communication Ideas

### DISCUSSION THREAD

**Discussion by andrewrynhard** [cosi-project/community#3] [Ideas]
- Matrix (We have #cosi-project:matrix.org already, thanks @Ulexus)
- Slack
- Forum (GitHub discussions works but putting this down for discussion)

**Comment by Ulexus** [comment]
Typo:  #cosi-project:matrix.org

**Reply by Ulexus** [reply]
Wow, typoed the typo correction.  Sheesh.  #cosi-project:matrix.org

**Reply by andrewrynhard** [reply]
Fixed!
